### PR TITLE
Travis CI integration, Updated README.md, Added .gitignore, and additional tests added

### PR DESCRIPTION
**Pull request by pavgup**
I'm quietly improving this plugin and here's a set of cleanup-related tasks that don't change the functionality of the plugin, but improve the meta-repo information.  I've got a bug that I want to isolate with the tests that I'm starting to include, but I've yet to isolate that, so in the interim, hopefully these updates help.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: ruby
+
+rvm:
+  - 2.1
+  - 2.2
```
Comment (repeatedly) [inline .travis.yml]: Could you add 2.3 and ruby-head?
And add ruby-head to allow_failures is better.

Comment (pavgup) [inline .travis.yml]: Definitely.  Done.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
-# Fluent::Plugin::Kafka
+# fluent-plugin-kafka, a plugin for [Fluentd](http://fluentd.org)
+
+[![Build Status](https://travis-ci.org/pavgup/fluent-plugin-kafka.svg?branch=master)](https://travis-ci.org/pavgup/fluent-plugin-kafka)
```
Comment (repeatedly) [inline README.md]: Use https://travis-ci.org/htgc/fluent-plugin-kafka instead.

Comment (pavgup) [inline README.md]: Done!

**Comment by repeatedly** [comment]
Thanks!

**Pull request merged**